src/opensuse/42.3/Dockerfile

```diff
@@ -98,3 +98,5 @@ ENV __PUBLISH_RID=ubuntu.14.04-x64
 # Define variables that the build uses to locate the liblldb.so and the related header.
 ENV LLDB_INCLUDE_DIR=/llvm/include \
     LLDB_LIB_DIR=/llvm/lib64
+
+RUN /sbin/ldconfig
```

Comment:
As @MichaelSimons usually suggests, it would be nice to attach this to the previous RUN to reduce the number of layers.

Reply:
Done.